src/html/elevations/elevations.html

```diff
@@ -1,4 +1,3 @@
-<div class="ds-elevation-remove"></div>
```

Comment:
@RachelRVasquez it seems like a nice thing to have and we could include it in the example to show the full range of flat to elevation-5.  Thoughts?